fcm-channel/src/main/java/io/fcmchannel/sdk/FcmClient.java

```diff
@@ -252,6 +269,10 @@ public static boolean hasFloatingPermission() {
             || (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M && Settings.canDrawOverlays(context));
     }
 
+    public static String getFcmToken() {
```

Comment:
This method was added because FCM token was removed from shared preferences.

Reply:
well done